### DISCUSSION TITLE
What is the `getInstance` equivalent for V5 tooltips?

### DISCUSSION THREAD

**Discussion by nikonthethird** [twbs/bootstrap#31586] [General] [answered]
You can get a existing Modal instance by calling [`getInstance`](https://v5.getbootstrap.com/docs/5.0/components/modal/#getinstance) on the modal's HTML element.

[Tooltips](https://v5.getbootstrap.com/docs/5.0/components/tooltips/#methods) on the other hand offer no such method. How do I get to the tooltip's instance when all I have is an element?

**Comment by Johann-S** [comment, marked as answer]
It seems we forgot to add this method in our documentation, but you can call `getInstance` on Tooltips too 👍
